camelot/view/controls/formview.py

```diff
@@ -182,9 +204,8 @@ def _layout_changed(self, index=None, start=None, end=None):
                     widget_mapper,
                     self.columns,
                     # Serialize the admin's form display again when the layout has changed, e.g. to pick up different tab labels with different locales.
-                    #self.form_display,
-                    self.admin.get_form_display()._to_bytes(),
-                    self.admin
+                    self.form_display,
```

Comment:
Dit probleem was eerder al opgelost dan?

Reply:
ik ben er nt in geslaagd het probleem te reproduceren, dus ik vermoed dat het idd 'onbewust' opgelost is.

???